book/invalidation.md

```diff
@@ -1530,7 +1521,7 @@ establishes a dependency for one layout field upon another. I like to
 think of each `f` as being scoped to its field's computation.
 
 We also need to compute the `x` position of a `TextLayout`. That can
-use the previous sibling's font, *x* position, and width:
+use the previous sibling's font, `x` position, and width:
```

Comment:
Same comment as in the other PR, that I think italics here is correct. (Or, code font is arguable, but only if we make `font` and `width` also code font.)